### PR TITLE
LPS-179179 Build Service

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**
*The upgrade process is missing, I'm working on it*

### CODE REVIEW

**Comment by gabrielwas** [comment]
Waiting for the performance test @mateussgn is working.

@mateussgn could you link the PR here, please? Thx

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/util/AggregationObjectFieldUtil.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+						objectFieldSettingsValues.get(
+							"objectRelationshipName"))));
+
+		ObjectDefinition relatedObjectDefinition =
+			objectDefinitionPersistence.findByPrimaryKey(
+				objectRelationship.getObjectDefinitionId2());
+
+		return objectFieldPersistence.fetchByODI_N(
+			relatedObjectDefinition.getObjectDefinitionId(),
+			String.valueOf(objectFieldSettingsValues.get("objectFieldName")));
```
Comment (mateussgn) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/util/AggregationObjectFieldUtil.java]: I think we inline the `objectRelationship.getObjectDefinitionId2()`, like this:
```java
return objectFieldPersistence.fetchByODI_N(
	objectRelationship.getObjectDefinitionId2(),
	String.valueOf(objectFieldSettingsValues.get("objectFieldName")));
```

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java [hunk 2]
```diff
@@ -141,12 +142,27 @@ public ObjectField addCustomObjectField(
...
 			ObjectFieldUtil.createIndex(
 				_currentConnection, objectFieldPersistence.getDataSource(),
-				AggregationObjectFieldUtil.getRelatedObjectField(
-					objectDefinition, _objectDefinitionPersistence,
-					objectFieldPersistence, objectFieldSettings,
-					_objectRelationshipPersistence),
+				objectFieldPersistence.fetchByODI_N(
+					relatedObjectDefinition.getObjectDefinitionId(),
+					String.valueOf(
+						objectFieldSettingsValues.get("objectFieldName"))),
```
Comment (mateussgn) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java:165]: We can inline `objectRelationship.getObjectDefinitionId2()`

**Comment by samuelromeiro** [comment]
Closing this PR for now.

**Pull request closed without merging**